R/api.R

```diff
@@ -65,9 +73,25 @@ metadata <- function() {
     get_option("influenza-1957", "Influenza 1957"),
     get_option("influenza-1918", "Influenza 1918 (Spanish flu)")
   )
-  pathogen_idx <- match("pathogen", response$parameters$id)
-  response$parameters$options[[pathogen_idx]] <- pathogen_options
-  response
+  pathogen_idx <- match("pathogen", param_ids)
+  response$parameters[[pathogen_idx]]$options <- pathogen_options
+
+  hospital_capacity_idx <- match("hospital_capacity", param_ids)
+  step <- response$parameters[[hospital_capacity_idx]]$step
```

Comment:
I don't think I've understood what `step` is doing here - is it mostly being used to round the capacity value to the nearest 100?